pkg/mock/generator.go

```diff
@@ -1,3 +1,6 @@
 package mock
 
 //go:generate minimock -g -i github.com/instill-ai/component/pkg/base.UsageHandler -o ./ -s "_mock.gen.go"
+//go:generate minimock -g -i ./../operator/pdf/v0.commandRunner -o ./ -s "_mock.gen.go"
+//go:generate minimock -g -i io.WriteCloser -o ./ -s "_mock.gen.go"
+
```

Comment:
@jvallesm 
Do you mean we need to remove the blank line?
If yes, I met this error if I do it.
`pkg/mock/generator.go:5: error reading pkg/mock/generator.go: unexpected EOF`

Do you happen to know about it?

Reply:
I meant removing the line, yes. I actually detected it because I ran `go generate` with the modification I suggested in the same file, so it should work. 🤔 I don't know what can be happening on your env but I don't feel too strongly about this so you can keep the blank line if you don't find a quick solution.